pkg/sqlstore/impl/system/store.go

```diff
@@ -401,11 +402,17 @@ func (s *SystemStore) SaveEVMEvents(ctx context.Context, events []tableland.EVME
 // It receives an optional fromHeight to only look for blocks after a block number. If null it will look
 // for blocks at any height.
 func (s *SystemStore) GetBlocksMissingExtraInfo(ctx context.Context, lastKnownHeight *int64) ([]int64, error) {
-	params := db.GetBlocksMissingExtraInfoParams{
-		ChainID:    int64(s.chainID),
-		FromHeight: lastKnownHeight,
+	var blockNumbers []int64
```

Comment:
I made a change here because `sqlc` was not able to generate the old query correctly:
```
SELECT DISTINCT e.block_number
FROM system_evm_events e 
WHERE e.chain_id=?1 AND 
      (?2 is null OR e.block_number>?2) AND
      NOT EXISTS(select* from system_evm_blocks b WHERE e.chain_id=b.chain_id AND e.block_number=b.block_number)
ORDER BY e.block_number
```

[I broke that query in two](https://github.com/tablelandnetwork/go-tableland/pull/382/files#diff-8b677e635e3f5c4e0e23b5723035b61bc814cd63074425f673d7d941e51958fdR11-R22) and depending if the `lastKnownHeight` is set or not I pick the right query.